src/main/resources/application.conf

```diff
@@ -2,3 +2,14 @@ akka {
   loglevel = INFO
   log-dead-letters-during-shutdown = off
 }
+
+# The server pipeline configuration to capture call metrics.
+squbs.pipeline.server.default {
+  pre-flow = sampleMetricsFlow
```

Comment:
@anilgursel Do you think there is any documentation gap leading us to add this both at `pre-flow` and `post-flow`?